f/zeus/fn_zeusInit.sqf

```diff
@@ -66,23 +79,26 @@ _unit assignCurator _curator;
 _curator setCuratorWaypointCost 0;
 {_curator setCuratorCoef [_x,0];} forEach ["place","edit","delete","destroy","group","synchronize"];
 
-// Check if F3 AI Skill Selector is active and assign corresponding event-handler
-if({!isNil _x} count ["f_param_AISkill_BLUFOR","f_param_AISkill_INDP","f_param_AISkill_OPFOR"] > 0) then {
-	_curator addEventHandler ['CuratorObjectPlaced',{
+// Add event-handler for setAISkill
+private _curatorEH = _curator getVariable ["f_curator_EH", -1];
+if (_curatorEH == -1) then {
+	_curatorEH = _curator addEventHandler ['CuratorObjectPlaced',{
```

Comment:
`addEventHandler` needs to be run on the Zeus' machine not the server. In the case of briefing assignment, the whole function is run on the server via `remoteExec`. For the Zeus slots the event handler also only appears to exist on the server for some reason (something weird with the way curator module initialize?)

Reply:
fn_zeusInit.sqf runs only on the server.
Yes curator module initialization is very weird.

I changed it so that the EH is now added locally and in case of the curator module, we wait until it is linked with its unit.